addons/l10n_mx/views/report_templates.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="l10n_mx_external_layout_standard" inherit_id="web.external_layout_standard">
+        <div t-field="company.partner_id" position="attributes">
+            <attribute t-if="company.country_id.code == 'MX'" name="t-options">{"widget": "contact", "fields": ["address", "name", "vat"], "no_marker": true}</attribute>
+        </div>
+        <div t-field="company.partner_id" position="before">
```

Comment:
Shouldn't this be done only for MX ? And why is it done only for this layout and not the others ?

Reply:
- This is done for the attribute above, but not for the `<div t-field="company.partner_id" position="before">...`

- I don't see 
```xml
<div t-field="company.partner_id" position="before">
    <div t-esc="company.country_id.code"/>
</div>
```

in the other layouts :thinking: